core/math/vector2.h

```diff
@@ -114,6 +114,34 @@ struct _NO_DISCARD_ Vector2 {
 	_FORCE_INLINE_ Vector2 bezier_interpolate(const Vector2 &p_control_1, const Vector2 &p_control_2, const Vector2 &p_end, const real_t p_t) const;
 	_FORCE_INLINE_ Vector2 bezier_derivative(const Vector2 &p_control_1, const Vector2 &p_control_2, const Vector2 &p_end, const real_t p_t) const;
 
+	_FORCE_INLINE_ Vector2 rotate_toward(Vector2 p_to, real_t p_rotation_delta, real_t p_length_delta = 0.0f) {
+		normalize();
```

Comment:
This should not be here. `rotate_toward` should be usable with non-normalized vectors, just like slerp.